projects/vdk-plugins/vdk-meta-jobs/src/vdk/plugin/meta_jobs/api/meta_job.py

```diff
@@ -16,6 +16,7 @@ class SingleJob:
     job_name: str
     team_name: str = None
     fail_meta_job_on_error: bool = True
+    arguments: dict = None
     depends_on: List[str] = field(default_factory=list)
```

Comment:
nit: What is the commonality of SingleJob and IStandaloneDataJob? It is strange that the DagJob isn't already relying on vdk-core API. What's the data job job top-level interface?
That question is out of scope for the current PR, yet I would like to revise it, in terms of refining the domain model.

Reply:
Well, the commonality seems to be the job name and job arguments. SingleJob has the team name fail_meta_job_on_error and the depends_on list, meaning that SingleJob, IMO, could rely on IStandaloneDataJob. I will discuss this topic with the DAG-involved personas.